examples/multi_channel_demo/chcheck/main.go

```diff
@@ -16,29 +19,24 @@ func main() {
 	testChannels := []string{"testch1", "testch2", "testch3"}
 	checkChannel := "checkch"
 
-	for {
-		for i := 0; i < 3; i++ {
+	for i := 0; i < 3; i++ {
```

Comment:
why not use the `for index, testChannel := range(testChannels)`,  that way if you add more strings by any reason to your `testChannels` if wouldn't necessarily break the demo.

I know this is a demo but considering it's what most people would look for when trying to understand what inspr is and how does it work, i think is a nice idea to tidy the code up.

Reply:
I can fix it, but it was not the scope, I was just adapting it to the new reader, the current code also has this look

Reply:
I can fix it, but it was not the scope, I was just adapting it to the new reader, the current code also has this look